extensions/frontend-tiles/src/FrontendTiles.ts

```diff
@@ -74,6 +61,8 @@ export interface MeshExports {
 export interface QueryMeshExportsArgs {
   /** The token used to access the mesh export service. */
   accessToken: AccessToken;
+  /** If defined, the iTwinId associated with the Mesh Export */
```

Comment:
It says "if defined" but it's a required property.

Reply:
The docs were wrong here, at one point we were considering having `iTwinId` be optional, but i didn't update the doc comment when we changed that. That fix is pushed now.